test/DynamoCoreWpfTests/ViewExtensions/GraphNodeManagerViewExtensionTests.cs

```diff
@@ -172,6 +172,7 @@ public void FilterFrozenItemsTest()
         /// Test if the number of Nodes containing Null or Empty List matches what is shown on the UI
         /// </summary>
         [Test]
+        [Category("Failure")]
```

Comment:
Marking this test as failure for now till we fix the issue. 
Task: https://jira.autodesk.com/browse/DYN-6657